src/audio/multiband_drc/multiband_drc.c

```diff
@@ -321,18 +321,42 @@ static int multiband_drc_cmd_set_value(struct processing_module *mod,
 }
 #endif
 
-static int multiband_drc_set_config(struct processing_module *mod, uint32_t config_id,
+static int multiband_drc_set_config(struct processing_module *mod, uint32_t param_id,
 				    enum module_cfg_fragment_position pos,
 				    uint32_t data_offset_size, const uint8_t *fragment,
 				    size_t fragment_size, uint8_t *response,
 				    size_t response_size)
 {
 	struct multiband_drc_comp_data *cd = module_get_private_data(mod);
+	struct comp_dev *dev = mod->dev;
 
-	comp_dbg(mod->dev, "multiband_drc_set_config()");
+	comp_dbg(dev, "multiband_drc_set_config()");
 
-#if CONFIG_IPC_MAJOR_3
-	/* Other control types are possible only with IPC3 */
+#if CONFIG_IPC_MAJOR_4
+	struct sof_ipc4_control_msg_payload *ctl;
```

Comment:
line 336 only used under line 339, can move to line 339

Reply:
I combined the pointer declare and set. OK now?